### PR TITLE
 Added code of daily Coding Exercise-25

### PR DESCRIPTION
**Pull request by anushreeSonak**

### CODE REVIEW

File: First/dailyExercises/ArrayCompare.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        }
+        return true;
+    }
+
+    public static void main(String[] args) {
+        Integer[] arrayFirst = {
+                1,
+                2,
+                3,
+                4
```
Comment (waghmaresuyog) [inline First/dailyExercises/ArrayCompare.java]: Use it into a single line

File: First/dailyExercises/ArrayCompare.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        };
+        Integer[] arrSecond = {
+                1,
+                2,
+                4,
+                4
```
Comment (waghmaresuyog) [inline First/dailyExercises/ArrayCompare.java]: same here

File: First/dailyExercises/ArrayCompare.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+        };
+        Integer[] arrThird = {
+                1,
+                2,
+                3,
+                4
+        };
+        String[] arrFourth = {
+                "Java",
+                "World"
```
Comment (waghmaresuyog) [inline First/dailyExercises/ArrayCompare.java]: add in single line

File: First/dailyExercises/ArrayCompare.java [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+        };
+        String[] arrFifth = {
+                "JavaScript",
+                "World"
+        };
+        String[] arrSixth = {
+                "Java",
+                "World"
```
Comment (waghmaresuyog) [inline First/dailyExercises/ArrayCompare.java]: same here

File: First/dailyExercises/ArrayCompare.java (hunk 1 above)
Comment (anushreeSonak) [reply First/dailyExercises/ArrayCompare.java]: DONE !

File: First/dailyExercises/ArrayCompare.java (hunk 2 above)
Comment (anushreeSonak) [reply First/dailyExercises/ArrayCompare.java]: DONE !

File: First/dailyExercises/ArrayCompare.java (hunk 3 above)
Comment (anushreeSonak) [reply First/dailyExercises/ArrayCompare.java]: DONE !

File: First/dailyExercises/ArrayCompare.java (hunk 4 above)
Comment (anushreeSonak) [reply First/dailyExercises/ArrayCompare.java]: DONE !

**Review by waghmaresuyog — APPROVED** [review]
Approving PR

**Review by prashantkawa — APPROVED** [review]

**Pull request merged**